docs/user-manual/index.md

```diff
@@ -1356,6 +1369,28 @@ group in its group map.  It updates membership in the user group,
 which indirectly updates the membership in the product
 configuration.
 
+### Working with Non-Email-based Usernames
+
+On the Adobe Admin Console, you can configure a domain to use email-based user login names or non-email-based usernames.  We often user the term Username based login to mean non-email-based-login.  Username-based login can be used when email addresses are expected to change often or your organization does not allow email addresses to be used for login.
+
+To configure User Sync to work with username logins, you need to set several additional configuration items.
+
+In the `connector-ldap.yml` file:
+
+- Set the value of `user_username_format` to a value like '{attrname}' where attrname names the directory attribute whose value is to be used for the user name.
+- Set the value of `user_domain_format` to a value like '{attrname}' if the domain name comes from the named directory attribute, or to a fixed string value like 'example.com'.
+
+When processing the directory, User Sync will fill in the username and domain values from those fields (or values).
+
+The values given for these configuration items can be a mix of string characters and one or more attribute names enclosed in curly-braces "{}".  The fixed characters are combined with the attribute value to form the string used in processing the user.
+
+Note that `user_username_format` configuration item will need to produce an email address if the federated domain is using email-based login. That can happen because the field itself contains an email address, or because you set the format string to something like `{attrname}@example.com`.
```

Comment:
I think you should remove this paragraph completely, because the whole premise of the section is "how to set up for username-based login", and this paragraph is about "how to set up for email-based login :).  If you want to talk about email-based login in this section, I would just add a paragraph at the very bottom that says:

If your Federated ID domain uses email-based login, then you should not define either of the parameters describe here: User Sync will automatically use the user's email value for SSO purposes.